### PR TITLE
new name of r-pkg-validation job

### PR DESCRIPTION
**Pull request by tomszosz**
# Pull Request

Fixes #269

```
there is an error
[Invalid workflow file: .github/workflows/validation.yaml#L74](https://github.com/insightsengineering/r.pkg.template/actions/runs/2153379295/workflow)
The workflow is not valid. .github/workflows/validation.yaml (Line: 74, Col: 12): Job 'upload-release-assets' depends on unknown job 'r-pkg-validation'.
```

https://github.com/insightsengineering/r.pkg.template/actions/runs/2153379295

### CODE REVIEW

File: .github/workflows/validation.yaml [hunk 1]
```diff
@@ -71,7 +71,7 @@ jobs:
 
   upload-release-assets:
     name: Upload report to release 🔼
-    needs: r-pkg-validation
+    needs: build-install-check
```
Comment (dinakar29) [inline .github/workflows/validation.yaml:74]: The only time you want to upload release assets is when the release exists, so this would be incorrect.

Comment (arkadiuszbeer) [inline .github/workflows/validation.yaml:74]:
```suggestion
    needs: [ build-install-check, r-pkg-validation]
```

Maybe?

**Review by arkadiuszbeer — APPROVED** [review]

File: .github/workflows/validation.yaml (hunk 1 above)
Comment (dinakar29) [reply .github/workflows/validation.yaml:74]: The better logic would be to identify if a release (not just the tag) exists and then upload this asset.

Comment (tomszosz) [reply .github/workflows/validation.yaml:74]: good point I will rollback it and change conditions

Comment (dinakar29) [reply .github/workflows/validation.yaml:74]: Please don't. I have already rolled back.

Comment (dinakar29) [reply .github/workflows/validation.yaml:74]: Rollback PR: https://github.com/insightsengineering/r.pkg.template/pull/76

**Pull request merged**